UnityProject/Assets/Scripts/GameModes/NukeOps.cs

```diff
@@ -35,8 +35,10 @@ protected override bool ShouldSpawnAntag(PlayerSpawnRequest spawnRequest)
 		//the ratio is too low
 		var existingNukeOps = PlayerList.Instance.AntagPlayers.Count;
 		var inGamePlayers = PlayerList.Instance.InGamePlayers.Count;
-		if ((inGamePlayers > 0 && existingNukeOps == 0) ||
-		    existingNukeOps < Math.Floor(inGamePlayers * nukeOpsRatio)) return true;
+		//Check if nuke device is on the map
+		Nuke isNukeOnMap = FindObjectOfType<Nuke>();

```

Comment:
You should override the IsPossible() method from the base GameMode.cs and add the nuke check there instead, otherwise the game mode will be chosen and antags will never spawn in.